turso/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "turso-example"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+axum = { version = "0.6.10", features = ["macros"] }
+shuttle-axum = { version = "0.19.0", path = "../../services/shuttle-axum" }
+shuttle-runtime = { version = "0.19.0", path = "../../runtime" }
+shuttle-turso = { version = "0.19.0", path = "../../resources/turso" }
```

Comment:
It would be great if these weren't path dependencies (you can use the patches in .cargo/config.toml for testing).